### ISSUE TITLE
Add documentation ;)

### ISSUE THREAD

**Issue by Aklakan** [AKSW/SPARQL2NL#7]
Would be interested to know how well it would work for faceted search (i.e. facete integration)

**Comment by antonkulaga** [comment]
Yeah, it is not clear how to use it in my code, getting started is neccesary
